src/SKUI/js/ui.button.js

```diff
@@ -45,3 +45,8 @@ Button.prototype.set_caption = function( value ) {
   this.control.text( value );
   return value;
 };
+
+Button.prototype.set_image = function( value ) {
+  this.control.append($('<img/>').attr( 'src', value ))
```

Comment:
Ha! didn't think of that, you could accidentally create a nice stack of images in the top-left corner!